CIScripts/JenkinsTesterDSC/JenkinsTester.ps1

```diff
@@ -0,0 +1,55 @@
+Configuration JenkinsTester {
+    param(
+        [Parameter(Mandatory=$true)][string[]]$MachineName
+    )
+
+    Import-DscResource -ModuleName PsDesiredStateConfiguration
+    Import-DscResource -ModuleName cChoco
+    Import-DscResource -ModuleName PowerShellModule
+
+    Node $MachineName {
+        WindowsFeature NetFrameworkCore {
+            Ensure    = "Present"
+            Name      = "NET-Framework-Core"
+        }
+
+        cChocoInstaller installChoco {
+            InstallDir = "C:\ProgramData\chocolatey"
+            DependsOn = "[WindowsFeature]NetFrameworkCore"
```

Comment:
I believe that cchoco doesn't depends on NetFrameworkCore. It seems to be required only by Jenkins service wrapper. Am I right ?